src/editor/utils/datainterface.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module core/editor/utils/datainterface
+ */
+
+/**
+ * Interface provides methods for setting and getting data to/from editor main root element of the model tree
+ * using {@link module:core/editor/editor~Editor#data data pipeline}.
+ *
+ * @mixin DataInterface
+ * @implements module:core/editor/utils/datainterface~Data
+ */
+const DataInterface = {
+	/**
+	 * Sets the data in the editor's main root.
+	 *
+	 * @method #setData
+	 * @param {*} data The data to load.
+	 */
+	setData( data ) {
+		this.data.set( data );
+	},
+
+	/**
```

Comment:
Definition of these should go into the interface, not to the mixin. The interface defines the API. The mixin implements it.